nav2_amcl/include/nav2_amcl/amcl_node.hpp

```diff
@@ -244,7 +244,7 @@ class AmclNode : public nav2_util::LifecycleNode
     std::shared_ptr<cmr_msgs::srv::GetStatus::Response> response);
 
   // Nomotion update control. Used to temporarily let amcl update samples even when no motion occurs
-  std::atomic<bool> force_update_{false};
+  std::atomic<bool> force_update_{true};
```

Comment:
I saw you upgraded it but isn't better to initially keep it as ```false```?

Reply:
I set it to false in a later commit.